.flake8

```diff
@@ -4,6 +4,7 @@ ignore =
 	E203, # whitespace before ':'
 	### Disabled because it makes the samples look worse
 	F841, # local variable is assigned but not never used
+	W503, # line break before binary operator
```

Comment:
Isn't this "Disabled because of conflicts with black"? Black is the one who decides formatting, not flake8.

@trym-b